model/QueueBroker/NewSqlQueueBroker.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoTaskQueue\model\QueueBroker;
+
+use common_persistence_Manager;
+use common_persistence_SqlPersistence;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver\Statement;
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Query\QueryBuilder;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Exception;
+use InvalidArgumentException;
+use oat\generis\Helper\UuidPrimaryKeyTrait;
+use oat\tao\model\taskQueue\Queue\Broker\AbstractQueueBroker;
+use oat\tao\model\taskQueue\Task\TaskInterface;
+use oat\taoTaskQueue\model\QueueBroker\storage\NewSqlSchema;
+use PDO;
+use Throwable;
+
+/**
+ * Storing messages/tasks in newSQl DB.
+ */
+class NewSqlQueueBroker extends AbstractQueueBroker
+{
+    use UuidPrimaryKeyTrait;
```

Comment:
This has been introduced in generis `12.8.0`,  manifest dependecy didn't match that.

Reply:
fixed